cmd/flarectl/README.md

```diff
@@ -9,26 +17,79 @@ You must set your API key and account email address in the environment variables
 ```
 $ export CF_API_KEY=abcdef1234567890
 $ export CF_API_EMAIL=someone@example.com
-$ flarectl
-NAME:
+
+$ flarectl:
+
    flarectl - Cloudflare CLI
 
 USAGE:
    flarectl [global options] command [command options] [arguments...]
-
+   
 VERSION:
-   2015.12.0
-
+   2016.4.0
```

Comment:
Want to make this match what you just bumped it to? :-)

Reply:
I do, yes! #arbitraryversionnumbers

(probably worth baking the hash/tag into the build via -ldflags once I get CI building binaries for this)